crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -771,6 +767,15 @@ pub struct EntityLocation {
     pub table_row: TableRow,
 }
 
+impl EntityLocation {
+    const INVALID: EntityLocation = EntityLocation {
```

Comment:
This (and EntityMeta::EMPTY) deserves a doc comment on how it's used, but I won't block on it.

Reply:
I added a doc for empty, pending, reserved and invalid entity, which I don't sure whether is accurate. Mind take a look?